src/signals/settings/roles/containers/RoleFormContainer/index.js

```diff
@@ -32,37 +38,54 @@ export const RoleFormContainer = ({
   },
   onPatchRole,
   onSaveRole,
+  showGlobalNotification,
+  onResetResponse,
   userCan,
 }) => {
   const { roleId } = useParams();
+  const location = useLocation();
+  const history = useHistory();
   const role = list.find(item => item.id === roleId * 1);
   const title = `Rol ${roleId ? 'wijzigen' : 'toevoegen'}`;
+  const redirectURL = {
+    pathname: location.referrer || routes.roles,
+    state: location.state,
+  };
+
+  useEffect(() => {
+    let message;
+    let variant = VARIANT_SUCCESS;
+    if (responseError) {
+      message = 'Er is iets mis gegaan bij het opslaan';
```

Comment:
This message is too generic to be displayed to a user; if we know what happened, the message should state what happened. This will also help us in the future with debugging the application and tracking errors. You can use the `errorMessageDictionary` in the `api` service to get an error message that is descriptive and context aware.

Reply:
ik zie nu dat de hele error message showGlobalNotification al in de saga wordt gedaan dus kan die hier in de container eruit. alleen de success is dan nog relevant